parser/prism/Translator.cc

```diff
@@ -81,6 +81,42 @@ bool isStringLit(const ast::ExpressionPtr &expr) {
     return false;
 }
 
+// Flattens the key/value pairs from the Kwargs Hash into the destination container.
+// If Kwargs Hash contains any splats, we skip the flattening and append the hash as-is.
+template <typename Container> void flattenKwargs(unique_ptr<parser::Hash> &kwargsHash, Container &destination) {
```

Comment:
I'm keeping this template to match what's already in `Desugar.cc`, even though there's only a single instantiation in this PR.

I suspect that eventually we'll be able to remove it, and treat both the "splatty" and "regular" Send desugar codepaths the same.